### DISCUSSION TITLE
It seems that your device does not support camera (or it is locked). The application will be closed

### DISCUSSION THREAD

**Discussion by 4EdgeIT** [Sunbird-Saral/Community#85] [Q&A]
While scanning sheet on my Samsung Galaxy Tab A (Android version 11), getting the error 

"It seems that your device does not support camera (or it is locked). The application will be closed"

What may be the solution for this?

**Comment by harish-challa** [comment]
Hello, We need further data to replicate this error. Can you please share with screenshots.

**Reply by 4EdgeIT** [reply]
@harish-challa Here is the screenshot of the error

![WhatsApp Image 2023-08-30 at 23 01 09](https://github.com/Sunbird-Saral/Community/assets/79844876/c967351b-8959-46ed-95a2-ec7db1e5a9df)

**Reply by harish-challa** [reply]
This has been fixed in Dev and expected to be pushed out in next release

**Reply by 4EdgeIT** [reply]
@harish-challa When will be this version moved to production?
